From 6515b020022ca6b14b61f3b7046495fc2582eb56 Mon Sep 17 00:00:00 2001
From: Kai Liu <kai.liu@suse.com>
Date: Wed, 25 May 2022 17:31:43 +0800
Subject: [PATCH] Revert "Linux 5.10.118"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit c204ee3350ebbc4e2ab108cbce7afc0cac1c407d.

Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f9210e43121d..8f611d79d5e1 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 118
+SUBLEVEL = 117
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.1

